CVE-2015-0232
Community BUG:
https://bugs.php.net/bug.php?id=68799
Community CODE:
http://git.php.net/?p=php-src.git;a=commit;h=55001de6d8c6ed2aada870a76de1e4b4558737bf 
This patch was created from the community reports above.


--- php-5.3.29/ext/exif/exif.c_orig	2015-06-14 15:28:49.975351692 -0700
+++ php-5.3.29/ext/exif/exif.c	2015-06-14 15:29:28.860406969 -0700
@@ -2722,7 +2722,7 @@
 static int exif_process_unicode(image_info_type *ImageInfo, xp_field_type *xp_field, int tag, char *szValuePtr, int ByteCount TSRMLS_DC)
 {
 	xp_field->tag = tag;	
-
+	xp_field->value = NULL;
 	/* Copy the comment */
 #if EXIF_USE_MBSTRING
 /*  What if MS supports big-endian with XP? */
